db/migrate/20200428092948_update_tubes_to_be_containers.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateTubesToBeContainers < ActiveRecord::Migration[6.0]
```

Comment:
Is there any risk of doing it like this? Could you get Harry to look at this just to make sure that this is going to work. We also have other team members with experience of migrating in production.

Reply:
I reviewed all the ways you can migrate data and this was the lowest risk I could find without introducing additional dependencies.  I read that using SQL directly and avoiding the model classes is the way forward.  Model classes might change name and this can break old migrations.  By referring to the tables directly, we know that table will always exist at this point in the migration history even if the model classes get renamed.